polybar-scripts/vpn-mullvad/README.md

```diff
@@ -0,0 +1,27 @@
+# Script: vpn-mullvad
```

Comment:
We should replace the existing script with your work.
Rename the script and directory to `vpn-mullvad-status`.